Tasks/AzurePowerShellV4/azurepowershell.ts

```diff
@@ -3,7 +3,7 @@ import path = require('path');
 import os = require('os');
 import tl = require('azure-pipelines-task-lib/task');
 import tr = require('azure-pipelines-task-lib/toolrunner');
-import * as telemetry from 'azure-pipelines-tasks-utility-common/telemetry';
+//import * as telemetry from 'azure-pipelines-tasks-utility-common/telemetry';
```

Comment:
Remove this. DOn't just comment if its not getting used

Reply:
As suggested removed the commented code.